spec/mailers/user_mailer_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe UserMailer, type: :mailer do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
Can you copy guts from this: https://github.com/WorkHands/RAPiDSkills/blob/welcome-email-91/spec/mailers/user_spec.rb and just add a minimal check or two for this case?